src/rgw/rgw_admin.cc

```diff
@@ -83,6 +85,7 @@ using namespace std;
 
 static rgw::sal::Driver* driver = NULL;
 static constexpr auto dout_subsys = ceph_subsys_rgw;
+ceph::async::io_context_pool context_pool;
```

Comment:
why is this global? putting it on the stack of `main()` would make it a lot easier to reason about order of initialization/destruction

Reply:
Thought it might need to be at the same scope as the driver, but turned out there's no need. Fixed.